projects/js-packages/components/components/text/stories/Text-MDX-Documentation.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Source, Story, Canvas } from '@storybook/addon-docs';
 import dedent from 'ts-dedent';
-import Text, { SPACING_VALUES, BOX_MODEL_VALUES, H2, H3, Title } from '../index.jsx';
+import Text, { BOX_MODEL_VALUES, H2, H3, Title } from '../index';
```

Comment:
minor and just FYI, the SPACING_VALUES name was inspired by [MUI](https://mui.com/system/spacing/)